VARIABLES.md

```diff
@@ -966,7 +966,7 @@ Default:  "{{ ssl_mutual_auth_enabled }}"
 
 ### kafka_connect_custom_rest_extension_classes
 
-Additional set of extension classes
+Additional set of Connect extension classes.
```

Comment:
shoot, actually make this change in the defaults/main.yml and then rerun the docs script